src/components/PathwayCourse/styles.js

```diff
@@ -104,6 +104,25 @@ const useStyles = makeStyles((theme) => ({
   progressBar: {
     width: "95%",
   },
+  certButtons: {
+    display: "flex",
+    justifyContent: "space-around",
+    marginTop: "32px"
+  },
+  greenButton: {
+    padding: "8px 16px !important",
+    borderRadius: "8px !important",
+    backgroundColor: "#48A145 !important",
+    color: "#FFFFFF !important",
```

Comment:
Reference theme colors here to allow for changes

Reply:
There are some MUI version issues in the project. So cannot reference the theme prop.